config.json

```diff
@@ -764,7 +764,7 @@
         "name": "Anagram",
         "uuid": "309fa4f1-e03a-4ab2-b371-cdf742501cf7",
         "practices": [],
-        "prerequisites": [],
+        "prerequisites": ["strings", "arrays", "basics", "for-loops", "conditionals", "comparison"],
```

Comment:
Because `basics` is the prerequisite of all "other things", it doesn't need to be listed here. 

Otherwise, great! Thanks :D

Reply:
Sure, removed it in https://github.com/exercism/javascript/pull/1548/commits/739d8f23adcb4eb75b39c6d3da18b308f69adda6